### ISSUE TITLE
Routes NOT WORKING. 

### ISSUE THREAD

**Issue by Kolovrat19** [leonardovilarinho/vue-acl#76]
Even in the example. 

`export default new AclCreate({
  initial: 'editor',
  notfound: '/pages/not-authorized',
  router,
  acceptLocalRules: true,
  globalRules: {
    admin: new AclRule('admin').generate(),
    editor: new AclRule('editor').or('admin').generate(),
    }
})

                {
                    path: '/admin',
                    name: 'admin',
                    component: () => import('./views/DashboardECommerce.vue'),
                    meta: {
                        rule: 'admin'
                    }
                },`
and to "site/admin" go without ploblem

**Comment by nongtiny** [comment]
Did you already initialize this acl file in the main.js file? and 
Did you set the rule of that user before entering that page?

**Comment by Kolovrat19** [comment]
Yes, I did it. In other projects, it does not work either. 
[https://pixinvent.com/demo/vuexy-vuejs-admin-dashboard-template/documentation/auth-and-acl/permissionControl.html#configuration](url)

**Comment by danijelh** [comment]
@Kolovrat19 Read the Warning part.

```
WARNING

We used vue-acl library for providing ACL in our application. Creator of vue-acl rewritten whole code since 4.0.7(approx). Since then route protection is not working.

So, we highly recommend till creator of that library don't fix that bug, Please use vue-acl@4.0.7.
```

Use the exact version 4.0.7. I really do hope creator of the library will fix this soon as possible as I am facing the same problem :/

**Comment by ErickZH** [comment]
same problem :( @leonardovilarinho

**Comment by leonardovilarinho** [comment]
Hi

I fixed this in version 4.1.6, it was a conditional that ended up making the use of mandatory middleware.

PS: The package is open source, so any developer can submitting PR and not just charging, usually when I'm notified of new PR I look faster, and if anyone wants to become a member of the project just get in touch.

**Comment by ErickZH** [comment]
😄 Thanks @leonardovilarinho
